src/Microsoft.SqlTools.ServiceLayer/LanguageServices/BindingQueue.cs

```diff
@@ -288,7 +288,7 @@ private void ProcessQueue()
                             CancellationTokenSource cancelToken = new CancellationTokenSource();
                      
                             // run the operation in a separate thread
-                            var bindThread = new Thread(() =>
+                            var bindTask = Task.Run(() =>
```

Comment:
Just pointing out that given the generic try/catch handling here, we don't need to use ContinueWithOnFaulted as it shouldn't do anything so this looks fine to me

Reply:
@kevcunnane that's what I thought.  I left the ContinueWithOnFaulted on the timeout continuation in case the Set() call threw..presumably from a nullref on the bindingcontext